torch/csrc/jit/graph_executor.cpp

```diff
@@ -104,73 +105,16 @@ struct ExecutionPlanAutogradFunction : public autograd::Function {
 // This will unwrap Variables, run the plan, and re-wrap them.
 // It can optionally also have a gradient which is hooked up
 // to the output Variables if present.
-struct ExecutionPlan {
-  ExecutionPlan(std::shared_ptr<Graph>& graph)
-      : f(graph),
-        graph(graph),
-        num_inputs(graph->inputs().size()),
-        num_outputs(graph->outputs().size()) {}
-  ExecutionPlan(std::shared_ptr<Graph>& graph, Gradient grad)
-      : f(graph),
-        graph(graph),
+struct DifferentiableGraphOp {
```

Comment:
we should rename ExecutionPlanAutogradFunction to DifferentiableGraphAutogradFunction, because it no longer has anything to do with ExecutionPlans.

Reply:
:+1: